src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp

```diff
@@ -176,7 +176,7 @@ CreatureAI* GetAI_mob_azaloth(Creature *_creature)
 
 enum 
 {
-    SPELL_SHADOWBOLT = 9613,
+    SPELL_SHADOWBOLT_9613 = 9613,

```

Comment:
yep, but in this case that solution here is alright, improves readability
-> didn't check for more SPELL_SHADOWBOLT